roles/rsyslog/defaults/main.yaml

```diff
@@ -362,7 +362,7 @@ rsyslog_conf_global_options:
     options: |-
       global(
         defaultNetstreamDriver="{{ rsyslog_default_netstream_driver }}"
-      {% if not rsyslog_default|default(true)|bool %}
+      {% if not rsyslog_default|default(false)|bool %}
```

Comment:
perhaps the use of the default filter could be eliminated completely, given that we have the variable in defaults. This way you have to maintain the default value at two places.